src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -488,6 +488,12 @@ def load_command_table(self, _):
         g.custom_command('unlink', 'unlink_user_function', validator=validate_staticsite_sku)
         g.custom_show_command('show', 'get_user_function', validator=validate_staticsite_sku)
 
+    with self.command_group('staticwebapp backends', custom_command_type=staticsite_sdk) as g:
+        g.custom_command('validate', 'validate_backend', validator=validate_staticsite_sku)
+        g.custom_command('link', 'link_backend', validator=validate_staticsite_sku)
+        g.custom_command('unlink', 'unlink_backend', validator=validate_staticsite_sku)
```

Comment:
```suggestion
        g.custom_command('validate', 'validate_backend', validator=validate_staticsite_sku, exception_handler=ex_handler_factory())
        g.custom_command('link', 'link_backend', validator=validate_staticsite_sku, exception_handler=ex_handler_factory())
        g.custom_command('unlink', 'unlink_backend', validator=validate_staticsite_sku, exception_handler=ex_handler_factory())
```


We should consider adding exception handlers to these commands. I tried to link a function app that was already linked to another SWA and got an unhelpful error. Without the exception handler, the command just fails with `Conflict` instead of printing the full error message (which was `Cannot link backend with a preexisting Azure Static Web Apps configuration.` in my case). I saw the same behavior with the validate command